webapp/src/components/propertyValueElement.tsx

```diff
@@ -48,9 +48,11 @@ const PropertyValueElement = (props:Props): JSX.Element => {
     const finalDisplayValue = displayValue || emptyDisplayValue
     const [open, setOpen] = useState(false)
 
+    const editableFields: Array<PropertyType> = ['text', 'number', 'email', 'url', 'phone']
+
     const saveTextProperty = useCallback(() => {
         if (editableFields.includes(props.propertyTemplate.type)) {
-            if (value !== props.card.fields.properties[props.propertyTemplate.id] || '') {
+            if (value !== (props.card.fields.properties[props.propertyTemplate.id] || '')) {
```

Comment:
@jespino @wiggin77 can ESLint catch such conditions inside `if`? It seems that there is no much sense in using additional `|| ''` with boolean expression.